src/summary/patient-chart-summary.css

```diff
@@ -5,3 +5,11 @@
 :global(.omrs-breakpoint-gt-phone) .patientSummary {
   padding: 1.25rem 1.5rem 0 1.5rem;
 }
+
+body {
+  margin: 0;
+}
+.patientSummary {
+  background-color: var(--omrs-color-bg-lowest-contrast);
+  min-height: 90vh;
```

Comment:
I don't like this -- it's better to just let content flow naturally and let scrolling be completely determined by content size and viewport height. Why do we need this?

Reply:
The patient chart should not have a different color than the [canvas color](https://github.com/openmrs/openmrs-esm-styleguide/blob/master/src/canvas/canvas.css#L3) that is set throughout the app.

I'll talk to Greg about this to change it. It doesn't make sense to have different canvas colors for each page.

Reply:
Hi, agreed doesn't make sense to change canvas color through app.

I've been using the omrs-bg-lowest-contrast as the default canvas color in the mockups - for patient chart and search.

The omrs-color-background-canvas is the color from the hackathon designs, but was replaced when we designed the stylguide.

I'm looking @ storybook atm - and the following are the hackathon color styles no longer in use:

Primary blue
--omrs-color-primary-blue
Text black
--omrs-color-text-black
Inactive grey
--omrs-color-inactive-grey
Inactive grey two
--omrs-color-inactive-grey-two
Background canvas
--omrs-color-background-canvas
Critical alert red
--omrs-color-critical-alert-red

Reply:
👍 I have created https://github.com/openmrs/openmrs-esm-styleguide/pull/48 which removes those colors from the styleguide.